app/scripts/models/reliers/relier.js

```diff
@@ -75,36 +107,20 @@ define(function (require, exports, module) {
           // query parameters and server provided data override
           // resume provided data.
           self.populateFromStringifiedResumeToken(self.getSearchParam('resume'));
+          // TODO - validate data coming from the resume token
+
+          self.importSearchParamsUsingSchema(QUERY_PARAMETER_SCHEMA, AuthErrors);
 
-          self.importSearchParam('service');
-          self.importSearchParam('preVerifyToken');
-          self.importSearchParam('uid');
-          self.importSearchParam('setting');
-          self.importSearchParam('entrypoint');
-          if (! self.has('entrypoint')) {
-            // FxDesktop declares both `entryPoint` (capital P) and
-            // `entrypoint` (lowcase p). Normalize to `entrypoint`.
-            self.importSearchParam('entryPoint', 'entrypoint');
+          // FxDesktop declares both `entryPoint` (capital P) and
+          // `entrypoint` (lowcase p). Normalize to `entrypoint`.
+          if (self.has('entryPoint') && ! self.has('entrypoint')) {
```

Comment:
nit: Edge case both `entryPoint` and `entrypoint` exist do nothing? Should one override the other?